Recrutify/Recrutify.Host/Controllers/CandidatesController.cs

```diff
@@ -88,5 +89,14 @@ public async Task<ActionResult<CandidateDTO>> GetByIdAsync(Guid id)
 
             return Ok(result);
         }
+
+        [Authorize(Policy = Constants.Policies.FeedbackPolicy)]
+        [HttpPut("testFeedback")]
+        public async Task<ActionResult> CreateTestFeedbackAsync(CreateBullFeedbackTestDTO testResult)
+        {
+            var userId = new Guid(HttpContext.User.Claims.FirstOrDefault(c => c.Type == "sub")?.Value);
```

Comment:
Create folder Providers, create IUserProvider, and implement method GetUserId; add extension method AddProviders and registrate it. 
Pass IUserProvider to Service, 
Controller doesn't contain any logic!